apps/frontend/src/app/PageTeam/CharacterDisplay/Tabs/TabOptimize/Components/BuildDisplayItem.tsx

```diff
@@ -73,6 +73,8 @@ type BuildDisplayItemProps = {
   disabled?: boolean
   extraButtonsRight?: JSX.Element
   extraButtonsLeft?: JSX.Element
+  mainStatAssumptionLevel: number
```

Comment:
So from my understanding, you are prop drilling these 2 props used by `BuildDisplayItem` from optConfig, so that any other changes to optconfig does not trigger rerendering of `BuildDisplayItem`, is that correct?

Reply:
Yep